lib/api/Cargo.toml

```diff
@@ -26,6 +26,7 @@ indexmap = { version = "1.6", features = ["serde-1"] }
 cfg-if = "1.0"
 thiserror = "1.0"
 more-asserts = "0.2"
+tracing = "0.1"
```

Comment:
can we have this as an optional dependency depending on a feature flag?

Reply:
done